dts: net: wireless: Simplify the description of the binding

rruuaanng opened the pull request:
Remove redundant descriptions in net/wireless bindings, such as "This is a representation of" and "... node".

kartben commented on dts/bindings/net/wireless/generic-fem-two-ctrl-pins.yaml:
```diff
@@ -2,8 +2,8 @@
 description: |
-    This is a representation of generic radio Front-End Module (FEM)
-    that has a two-pin control interface (CTX, CRX).
+    generic radio Front-End Module (FEM) that has a two-pin
```
```suggestion
    Generic radio Front-End Module (FEM) with a two-pin
```

rruuaanng committed:
dts: net: wireless: Simplify the description of the binding

Remove redundant descriptions in net/wireless bindings, such
as "This is a representation of" and "... node".

Signed-off-by: James Roy <rruuaanng@outlook.com>
